osidb/models/flaw/cvss.py

```diff
@@ -65,13 +65,13 @@ def _validate_rh_cvss3_and_impact(self, **kwargs):
                 self.alert(
                     "zero_CVSSv3_score_with_set_impact",
                     "RH CVSSv3 score must not be zero if flaw impact is set.",
-                    **kwargs
```

Comment:
If you have multiple commits where one fixes the other, I would suggest to squash them. It makes the PR easier to grasp from the reviewer point of view and can also later ease the understanding of the git history if we for some reason need to look back for something.